src/App.js

```diff
@@ -1,12 +1,61 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import './App.css';
+import axios from 'axios';
+import PokemonCard from "./PokemonCard";
+import fatPikachu from './assets/kindpng_468972.png'
+import pokeLogo from './assets/kindpng_24249.png'
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+
+    const [pokemon, setPokemon] = useState([])
```

Comment:
Zodra de setter aangeroepen wordt dan is het geen array mee. Ik zou dan de default waarde `undefined` maken, dan kan je check onderin wat simpeler
```suggestion
    const [pokemon, setPokemon] = useState()
```